index.html

```diff
@@ -1,176 +1,780 @@
-            <!-- <button class="tablinks" onclick="openTab(event, 'News')">News</button>-->
-            <button class="tablinks" onclick="openTab(event, 'Foodprint')">Foodprint</button>
-            <button class="tablinks" onclick="openTab(event, 'Team')">Team</button>
-            <button class="tablinks" onclick="openTab(event, 'Contact')">Contact</button>
-        </div>
-
-        <div id="About" class="tabcontent">
-            <p> It's time to quantify our carbon footprints.</p>
-            <p>Sign up to our mailing list to follow the launch of our app, Foodprint.</p>
-            <div id="mc_embed_signup">
-                <form
-                    action="https://quanteco.us5.list-manage.com/subscribe/post?u=84b372b9598d22040af9583e0&amp;id=0def27f7e4"
-                    method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank">
-
-                    <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-                        <input class="mdl-textfield__input" type="email" value="" name="EMAIL" id="mce-EMAIL"
-                            data-required>
-                        <label class="mdl-textfield__label" for="mce-EMAIL">Email address</label>
-                    </div>
 
-                    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-                    <div style="position: absolute; left: -5000px;" aria-hidden="true">
-                        <input type="text" name="b_84b372b9598d22040af9583e0_0def27f7e4" tabindex="-1" value="">
-                    </div>
+  <!-- ***** Preloader Start ***** -->
+  <div id="js-preloader" class="js-preloader">
+    <div class="preloader-inner">
+      <span class="dot"></span>
+      <div class="dots">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </div>
+  </div>
+  <!-- ***** Preloader End ***** -->
 
-                    <button type="submit" name="subscribe" id="mc-embedded-subscribe"
-                        class="mdl-button mdl-button--raised mdl-button--primary">
-                        Subscribe
-                    </button>
-            </div>
-            </form>
-        </div>
+  <!-- ***** Header Area Start ***** -->
+  <header class="header-area header-sticky wow slideInDown" data-wow-duration="0.75s" data-wow-delay="0s">
+    <div class="container">
+      <div class="row">
+        <div class="col-12">
+          <nav class="main-nav">
+            <!-- ***** Logo Start ***** -->
+            <a href="index.html" class="logo">
+              <img src="assets/images/qe/quanteco2.png" alt="Quanteco" style="width: 130px;">
+            </a>
+            <!-- ***** Logo End ***** -->
+            <!-- ***** Menu Start ***** -->
+            <ul class="nav">
+              <li class="scroll-to-section"><a href="#top" class="active">Login</a></li>
```

Comment:
I would very much remove the focus on "Login". There is already a big shiny green "To The Alpha" button, so why bother with "Login"?